elliott

```diff
@@ -361,13 +367,16 @@ described below:
 
     Note: Using --id without --add is basically pointless
 
-AUTOMATIC: For this use-case the --group option MUST be provided. The
+SWEEP: For this use-case the --group option MUST be provided. The
 --group automatically determines the correct target-releases to search
 for MODIFIED bugs in.
 
-MANUAL: The --group option is not required if you are specifying bugs
+LIST: The --group option is not required if you are specifying bugs
```

Comment:
Maybe add words like `In LIST mode you must provide a list of IDs to attach with the --id option`